store/rdbms/rdbms_schema.go

```diff
@@ -238,6 +239,23 @@ func (Schema) Labels() *Table {
 	)
 }
 
+func (Schema) Templates() *Table {
+	return TableDef("templates",
+		ID,
+		ColumnDef("rel_owner", ColumnTypeIdentifier),
+		ColumnDef("handle", ColumnTypeVarchar, ColumnTypeLength(handleLength)),
+		ColumnDef("language", ColumnTypeText),
+		ColumnDef("type", ColumnTypeText),
+		ColumnDef("partial", ColumnTypeBoolean),
+		ColumnDef("meta", ColumnTypeJson),
+		ColumnDef("template", ColumnTypeText),
```

Comment:
Are we safe to assume that template will always be text?
Would we be better of with wrapping it into struct/doc and save it as json?

Reply:
I... don't know...

We will be implementing some sort of a version control system for our templates so this part will need to be reworked. I didn't find much sense in complicating this part (yet) as only HTML and plaintext templates will be supported for now (plain text stuff).

I would leave it as is for now, and we can overengineer the structure a bit later when we need to.